templates/default/snapraid.conf.erb

```diff
@@ -6,11 +9,11 @@
 # Default configuration borrowed from: http://zackreed.me/articles/72-snapraid-on-ubuntu-12-04
 
 <% node['snapraid']['parity_disks'].each_with_index  do |parity, index| %>
-<%=index > 0 ? "#{index+1}-" : ''%>parity <%= parity %>/snapraid.parity
+<%=index > 0 ? "#{index+1}-" : ''%>parity <%= parity + path_separator%>snapraid.parity
```

Comment:
Oh silly ruby. I thought the whole reason for that method was for windows. If you are feeling up to it, you could try this:

```erb
<%= parity + File::SEPARATOR%>snapraid.parity
```

Should work, but I don't have windows to test it on.